aptos-move/mvhashmap/src/versioned_data.rs

```diff
@@ -278,6 +291,10 @@ impl<K: Hash + Clone + Debug + Eq, V: TransactionWrite> VersionedData<K, V> {
         use ValueWithLayout::*;
         match v.versioned_map.entry(ShiftedTxnIndex::zero_idx()) {
             Vacant(v) => {
+                if let Some(base_size) = value.bytes_len() {
```

Comment:
I would assume these counters have no visible overhead? Do we want them to be part of implementation or enabled conditionally?

Reply:
I want them always, otherwise they are useless